int32 batch_id
---
bool success
